app/helpers/quadicon_helper.rb

```diff
@@ -224,7 +221,7 @@ def img_tag_reflection
   end
 
   # Renders a quadicon for PhysicalServer
-  #
+  # TODO: use quadicon decorator
```

Comment:
what about all the todos? is it for followup PR? to be fixed by provider teams?

Reply:
It's to mark methods which can use quadicon decorator. I can refactor them now, or in follow up PR. Decision is on @martinpovolny or @skateman.

Reply:
Marking the methods is ok. We cannot do all cleanups at once. Thx!